src/main/java/frc/robot/constants/RobotConstants.java

```diff
@@ -10,7 +10,7 @@
 
     public TesterConstants testerConstants = new TesterConstants();
     public VisionConstants visionConstants = new VisionConstants();
-
+    public LocalConstants localConstants = JsonHandler.getConstants();
```

Comment:
this doesn't work with dependency injection very well because anything that uses this gains access to a ton of values it has no connection to

Reply:
this is separated into sub-classes specific to commands or subsystems.